sources/packages/web/src/views/aest/Login.vue

```diff
@@ -1,41 +1,58 @@
 <template>
-  <v-card elevation="2" class="mx-auto mt-15" max-width="690px" outlined>
-    <v-card-header>
-      <v-card-header-text>
-        <v-card-title class="my-6 bold-text" style="font-size: 27px"
-          >Welcome to AEST Portal</v-card-title
-        >
-        <v-card-subtitle>Welcome text goes here…</v-card-subtitle>
-      </v-card-header-text>
-    </v-card-header>
-    <v-card-text
-      ><div>
-        Please use your IDIR to authenticate. You must be previously authorized
-        by the system administrator in order to successfully login.
-      </div>
-      <banner
-        class="mt-2"
-        :type="BannerTypes.Error"
-        v-if="showNotAllowedUser"
-        summary="The user was validated successfully but is not currently allowed to
+  <full-page-container layout-template="centered-card" :enable-max-width="true">
```

Comment:
What is the idea behind using `:enable-max-width="true"`?
Is it to apply a fixed `730px` max-width? Is it something common for every `full-page-container` that will need a max-width defined?
For the login pages, do we need to use the `full-page-container`?
@ann-aot @dheepak-aot

Reply:
I believe the idea is to ensure a max-width(730 px) for full-page-container with fluid(full width) false. 
But I think the max-width class can be set based on fullWidth as false.

Reply:
The `full-page-container` already has a max-width class that is used when it needs to be centered (`full-page-container-size`).
This one is creating a new max-width defined as 730px which I guess was used only for login pages so far.
The question is: Is `enable-max-width` something generic with the potential to be used anywhere else? Do we have other possible places that will need to use the 730px?

Reply:
I went further and saw other pages. It seems like enable-max-width is only specific to login page.(And must not be used on other pages where fluid is false as it is breaking them). @ann-aot can surely give a better answer.

Reply:
yes the idea of `enable-max-width` is to apply `730px` max-width for login cards as discussed with @hellolynn-tbtb. As discussed with @andrewsignori-aot, we can avoid using `full-page-container` for login pages @guru-aot, instead, we can use `<v-container` with `v-cards` with `max-width = "730px"`.

Reply:
In that case for we create a common component(e.g. login-container) to use in all login pages. 
If not in this PR, you can also create the common component in next PR @guru-aot .